packages/principal/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@ipld/dag-ucan": "^3.3.2",
     "@noble/ed25519": "^1.7.3",
-    "@ucanto/interface": "workspace:^",
+    "@ucanto/interface": "workspace:^8",
```

Comment:
I don't think anything here was affected so not sure we need to force update

Reply:
I think you're right - I'm going to just go ahead and release it to make the upgrade a little easier for downstream folks (ie, "update everything to 8" rather than "update almost everything to 8")